network-setup.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+
+docker-compose run --rm wpcli core multisite-install \
+	--url="localhost" \
+	--base="/" \
+	--title="MIT Libraries" \
+	--admin_email="mjbernha@mit.edu" \
+	--admin_password="abcd1234"
+
+# Site 1, the parent site, comes for free
+
+# Site 2 - Dewey (deleted)
+docker-compose run --rm wpcli \
+	site create --url="localhost" --slug=_dewey   --title="Dewey"    --private
+docker-compose run --rm wpcli \
+    site delete --slug=_dewey --yes
+
+# Site 3 - Exhibits
+docker-compose run --rm wpcli \
+	site create --url="localhost" --slug=exhibits --title="Exhibits"
+
+# Site 4 - deleted
+docker-compose run --rm wpcli \
+	site create --url="localhost" --slug=_four --title="Four"
+docker-compose run --rm wpcli \
+    site delete --slug=_four --yes
+
+# Site 5 - MIT History
+docker-compose run --rm wpcli \
+	site create --url="localhost" --slug=mithistory --title="MIT History"
+
+# Site 6 - deleted
```

Comment:
What's up with having deleted sites? Is it to hit the magic number 7 below?

Reply:
Yes, unfortunately. Once we get this refactored, it shouldn't be necessary:

https://github.com/MITLibraries/MITlibraries-parent/blob/master/lib/news.php#L14